indy_node/test/anon_creds/test_send_write_permission_anyone.py

```diff
@@ -0,0 +1,88 @@
+import json
+import pytest
+from indy_node.test.helper import sdk_add_attribute_and_check
+from plenum.test.pool_transactions.helper import sdk_add_new_nym
+from plenum.test.helper import sdk_sign_request_from_dict, sdk_send_and_check, \
+    sdk_get_and_check_replies, sdk_sign_and_submit_req
+from plenum.common.exceptions import RequestRejectedException
+
+from indy_common.auth import Authoriser
+from indy.anoncreds import issuer_create_schema
+from indy.ledger import build_schema_request
+
+from indy_node.test.anon_creds.conftest import claim_def
+from indy_client.test.test_nym_attrib import attributeData, attributeName, attributeValue
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    # We need to reset authorization map to set new authorization rules
+    Authoriser.auth_map = None
+    OLD_ANYONE_CAN_WRITE = tconf.ANYONE_CAN_WRITE
+    tconf.ANYONE_CAN_WRITE = True
+
+    yield tconf
+
+    tconf.ANYONE_CAN_WRITE = OLD_ANYONE_CAN_WRITE
+    Authoriser.auth_map = None
+
+
+def test_client_can_send_nym(looper,
+                             txnPoolNodeSet,
+                             sdk_wallet_client,
+                             sdk_wallet_trust_anchor,
+                             sdk_pool_handle,
+                             attributeData):
+    # client can create another client NYM when ANYONE_CAN_WRITE set to True
+    sdk_new_client_wallet = sdk_add_new_nym(looper, sdk_pool_handle, sdk_wallet_client)
+    _, new_client_did = sdk_new_client_wallet
+
+    # client as an owner can add attribute to his NYM
```

Comment:
It's better to move the checks for ATTRIBs into a separate test method. Keep tests as small as possible.

Reply:
Ok